templates/category/index.html

```diff
@@ -1,30 +1,205 @@
 {% extends "base.html" %}
-{% load slice from shop %}
+{% load i18n %}
+{% load shop %}
 {% load staticfiles %}
 {% load bootstrap_pagination from bootstrap3 %}
 {% load render_bundle from webpack_loader %}
+{% load prices_i18n %}
+{% load product_first_image from product_images %}
 
 {% block footer_scripts %}
   {{ block.super }}
-  {% render_bundle 'category' 'js' %}
 {% endblock footer_scripts %}
 
 {% block title %}{{ category }} — {{ block.super }}{% endblock %}
 
 {% block breadcrumb %}
-    {{ block.super }}
-    {% for breadcrumb in breadcrumbs %}
-        <li{% if forloop.last %} class="active"{% endif %}>
-            {% if not forloop.last %}
-                <a href="{{ breadcrumb.get_absolute_url }}">{{ breadcrumb }}</a>
-            {% else %}
-                <span>{{ breadcrumb }}</span>
-            {% endif %}
-        </li>
-    {% endfor %}
+  {{ block.super }}
+  {% for breadcrumb in breadcrumbs %}
+    <li{% if forloop.last %} class="active"{% endif %}>
+      {% if not forloop.last %}
+        <a href="{{ breadcrumb.get_absolute_url }}">{{ breadcrumb }}</a>
+      {% else %}
+        <span>{{ breadcrumb }}</span>
+      {% endif %}
+    </li>
+  {% endfor %}
 {% endblock breadcrumb %}
 
 {% block content %}
-    {% csrf_token %}
-    <div id="category-page" data-category='{"id":{{category.id}}}'></div>
+  <div id="category-page">
+    <div class="category-top">
+      <div class="row">
+        <div class="col-md-7">
+          <ul class="breadcrumbs list-unstyled d-none d-md-block">
+            <li><a href="/">Home</a></li>
+            {% for ancestor in category.get_ancestors %}
+              <li><a href='{{ ancestor.get_absolute_url }}'>{{ ancestor.name }}</a></li>
+            {% endfor %}
+            <li><a href='{{ category.get_absolute_url }}'>{{ category.name }}</a></li>
+          </ul>
+        </div>
+        <div class="col-md-5">
+          <div class="row">
+            <div class="col-6 col-md-2 col-lg-6 filters-menu">
+              <span class="filters-menu__label d-sm-none">Filters</span>
+            </div>
+            <div class="col-6 col-md-10 col-lg-6">
+              <div class="sort-by">
+                <div class="click-area d-none"></div>
+                <button class="btn btn-link">
+                  <div>
+                    <span>
+                      Sort by:&nbsp;
+                      <strong>
+                        {% if request.GET.sort_by == 'name' or request.GET.sort_by == '-name' %}
+                          {% trans "name" context "Product name" %}
+                        {% elif request.GET.sort_by == 'price' or request.GET.sort_by == '-price' %}
+                          {% trans "price" context "Product price" %}
+                        {% endif %}
+                      </strong>
+                    </span>
+                    <div class="sort-order-icon">
+                      {% if request.GET.sort_by == '-name' or request.GET.sort_by == '-price' %}
```

Comment:
Here also a template tag could be used to return proper sorting icon.

Reply:
Done